library/test/src/lib.rs

```diff
@@ -290,7 +291,11 @@ where
     fn calc_timeout(timeout_queue: &VecDeque<TimeoutEntry>) -> Option<Duration> {
         timeout_queue.front().map(|&TimeoutEntry { timeout: next_timeout, .. }| {
             let now = Instant::now();
-            if next_timeout >= now { next_timeout - now } else { Duration::new(0, 0) }
+            if next_timeout >= now {
+                next_timeout - now
+            } else {
+                Duration::new(0, 0)
+            }
```

Comment:
I think you have been formatting with a stable rustfmt that doesn't support some of the options specified in `rustfmt.toml`. Please use `./x.py fmt` instead.